spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -40,8 +40,8 @@ the MicroProfile services the token will be accessed with.
 #### Required MP-JWT Headers
 
 alg:: This JOSE header parameter identifies the cryptographic algorithm used to secure the JWT.
-    - RSASSA-PKCS1-v1_5 SHA-256 or ECDSA using P-256 and SHA-256 algorithm are required when the claims have to be signed and must be specified as either "RS256", https://tools.ietf.org/html/rfc7518#section-3.3[RFC7518, Section 3.3] or "ES256", https://tools.ietf.org/html/rfc7518#section-3.4[RFC7515, Section 3.4].
-    - RSAES using Optimal Asymmetric Encryption Padding algorithm is required when the claims or nested JWT tokens have to be encrypted. It is used for encrypting the content encryption key and must be specified as "RSA-OAEP", https://tools.ietf.org/html/rfc7518#section-4.3[RFC7518, Section 4.3].
+    - RSASSA-PKCS1-v1_5 SHA-256 or ECDSA using P-256 and SHA-256 algorithm are required when the claims have to be signed and may be specified as either "RS256", https://tools.ietf.org/html/rfc7518#section-3.3[RFC7518, Section 3.3] or "ES256", https://tools.ietf.org/html/rfc7518#section-3.4[RFC7515, Section 3.4].  Support for the other asymmetric signature algorithms such as RS512, ES512 and others is optional.
+    - RSAES using Optimal Asymmetric Encryption Padding algorithm is required when the claims or nested JWT tokens have to be encrypted. It is used for encrypting the content encryption key and may be specified as either "RSA-OAEP" or "RSA-OAEP-256", https://tools.ietf.org/html/rfc7518#section-4.3[RFC7518, Section 4.3].  Support for the other key management algorithms such as "RSA-OAEP-384", "RSA-OAEP-512" and others is optional.
```

Comment:
I'm not sure what this means, where the algorithm may be specified as either the suggested 2 values but then all other algorithms can also be optionally supported.
And as I said - unfortunately it is a breaking change for 2.1.